src/scripts/Player.ts

```diff
@@ -363,7 +347,7 @@ class Player {
         }
 
         for (let pokemon of this._caughtPokemonList()) {
-            if (pokemon.levelObservable() < (this.gymBadges.length + 2) * 10) {
+            if (pokemon.levelObservable() < (App.game.badgeCase.badgeCount() + 2) * 10) {
```

Comment:
Was this intended to limit pokemon to level 20 before the first badge? I noticed a while ago that you could reach level 30 but never bothered to find out why. It is because we are now counting BadgeCase.Badge.None as a badge, so badgeCount before any real badges is already 1. Not sure if we want to change this line, or change badgeCount to make more sense.

Reply:
`BadgeCase.Badge.None` is kind of counterintuitive anyway, I'll try to change it so it's no longer needed.